src/XmlSerializationVisitor.php

```diff
@@ -179,14 +179,26 @@ public function visitInteger(int $data, array $type)
      */
     public function visitDouble(float $data, array $type)
     {
+        $dataResult = $data;
         $percision = $type['params'][0] ?? null;
-        if (is_int($percision)) {
+        if ($percision) {
```

Comment:
What about precision that equals `0`? Can we add test case to make sure that it works as previously?

Reply:
you are right, my error